### ISSUE TITLE
Suggest some new features

### ISSUE THREAD

**Issue by KlausMikhaelson** [KlausMikhaelson/wrestle-with-shapes#23]
Would love to see your imagination so just lmk if you have any new ideas to add in the game !

**Comment by MahirMahdi** [comment]
Add some new shapes, arenas and a special ability for each shape to spice it up a bit(Like a shape's special ability can be shield and another's can be thrust)but just for a short period.

**Comment by KlausMikhaelson** [comment]
@MahirMahdi sounds great, let's take one at a time, what would you like to see first ? and would really appreciate if you can create an issue for it !

**Comment by MahirMahdi** [comment]
Yes sure. Let's take one at a time then. 😀

Let me create an issue for the feature that I want to see first, then we can discuss about the other features after working on it.

**Comment by iburn78** [comment]
I just played the game by myself with both hands and found it quite fun. However, I guess if I played it with another person, it might be endless.

I think adding features that could easily end the game, such as a jump feature, would be beneficial. For example, when the opponent is pushing you to the edge, you could jump to make them fall off the edge.

**Comment by KlausMikhaelson** [comment]
@iburn78 so by that you mean making it online multiplayer right ? actually we can do that, few weeks ago we built https://sharktanks.tech/ which is an online multiplayer game so that could be a great idea I believe.

jumping sounds good to me, would you like to create a new issue for it ?

**Comment by iburn78** [comment]
@KlausMikhaelson Yes, eventually it could be an online multiplayer game. https://sharktanks.tech/ also looks fun. =) 

Let me create a new issue for adding a jump feature in the game.

**Comment by Vee339** [comment]
Hi, I have just played the game and it is great. 
One feature that is coming to my mind is that there should be a timer so that one can know in how much time they lost the game.

**Comment by harisdev-netizen** [comment]
After reviewing the "wrestle-with-shapes", here are some potential new features that could be added to enhance the functionality and user experience:

- Shape recognition: Implementing shape recognition would be a useful addition, allowing users to identify the type of shape that they are working with. This could be particularly helpful when working with more complex shapes, where it might be difficult to distinguish between different types of shapes.
- Interactive interface: Currently, the library is used via code. Adding an interactive interface, like a graphical user interface (GUI), would make it more accessible to users who are not comfortable writing code. This could also enable more complex operations to be performed, like grouping and ungrouping shapes, or dragging and dropping shapes.

Adding these features would enhance the functionality and versatility of the "wrestle-with-shapes", making it more useful for a wider range of applications.

**Comment by Helixjoe** [comment]
You could add a Hit feature where the shape can dash into the other shape making the other shape fly away
